src/lib/util/value.c

```diff
@@ -1502,14 +1510,20 @@ ssize_t fr_value_box_from_network(TALLOC_CTX *ctx,
 
 	switch (type) {
 	case FR_TYPE_STRING:
-		if (fr_value_box_bstrndup(ctx, dst, enumv, (char const *)src, len, tainted) < 0) {
+		if (fr_value_box_bstrndup(ctx, dst, enumv,
```

Comment:
You'll need to add a dbuff variant of `fr_value_box_bstrndup`

Reply:
Done and pushed; at least for now, `fr_value_box_bstrndup()` is the original and `fr_value_box_bstrndup_dbuff()` is the wrapper, because there are calls that pass NULL, and `FR_DBUFF_TMP(NULL, 0)` causes complaints from the address sanitizer.